README.md

```diff
@@ -133,20 +133,21 @@ You can also export FHIR resources from a FHIR server using [Bulk Export](https:
 > $BatchAccountName = "BatchAccountName. New batch account would be created if account name is null or empty."
 > $BatchAccountPoolName = "BatchAccountPoolName"
 > $BatchComputeNodeSize = "Node size for batch node. Default value is 'Standard_d1'"
+> $ResourceGroupName = "Resource group name for Data Factory. Default value is 'data_factory_name+resourcegroup'"

```

Comment:
Better to be $dataFactoryName + "resourcegroup" ?

Reply:
Fixed